.github/scripts/changelog_checker

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash
```
(if it's going to run with bash, it would be better for that to be reflected in the header!)